src/spv/inst-make.cpp

```diff
@@ -1210,6 +1210,29 @@ bool Instruction::makeResult(
     case spv::OpLabel: // 248
         data[result_at].redefine(new Primitive(location));
         break;
+    case spv::OpTypeRayQueryKHR: { // 4472
+        data[result_at].redefine(new Type(Type::rayQuery()));
+        break;
+    }
+    case spv::OpTypeAccelerationStructureKHR: // 5341
+        // Will define the type in class "AccelerationStructureManager" in "acceleration-structure.cxx" in "copyFrom()"
+        // method. Passing in empty vectors instead.
+        data[result_at].redefine(
+                new Type(Type::accelerationStructure(std::vector<const Type*> {}, std::vector<std::string> {})));
+        break;
+    case spv::OpConvertUToAccelerationStructureKHR: // 4447
+    case spv::OpRayQueryProceedKHR: // 4477
+    case spv::OpRayQueryGetIntersectionTypeKHR: // 4479
+    case spv::OpReportIntersectionKHR: // 5334
+    case spv::OpRayQueryGetRayTMinKHR: // 6016
+    case spv::OpRayQueryGetRayFlagsKHR: // 6017
+    case spv::OpRayQueryGetWorldRayDirectionKHR: // 6029
+    case spv::OpRayQueryGetWorldRayOriginKHR: { // 6030
+        assert(hasResultType);
+        Type* ret_type = getType(0, data);
+        data[result_at].redefine(ret_type->construct());
```

Comment:
Are these instructions used in your examples? If there is a place to put TODO, it would probably be here (to note that this code only generates dummy values). However, I would prefer you to get rid of half-implemented features.